test/integration/wdio.conf.js

```diff
@@ -322,7 +317,7 @@ const wdioConfig = {
     // and 30 processes will get spawned. The property handles how many capabilities
     // from the same test should run tests.
     //
-    maxInstances: 10,
+    maxInstances: 1,
```

Comment:
It'd be really nice to have this run parallel, but those runs seem to clobber each other. I suspect shenanigans with cookies getting globally added and removed, and communication with the single narrative application we have running. I think it's worth trying to fix that by ensuring that each test spec only uses a single narrative, but this PR's just about returning to status quo.